testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketSslClientRenegotiateTest.java

```diff
@@ -121,6 +121,8 @@ public SocketSslClientRenegotiateTest(
 
     @Test(timeout = 30000)
     public void testSslRenegotiationRejected() throws Throwable {
+        // BoringSSL does not support renegotiate by purpose.
```

Comment:
Same issue with the comment here ... "BoringSSL intentionally does not support renegotiation"?